src/components/home/InformationPanel.vue

```diff
@@ -0,0 +1,44 @@
+<script lang="ts" setup>
+import LineDescriptions from './LineDescriptions.vue'
+import TravelTimes from './TravelTimes.vue'
+import UiTrambusTitle from '../ui/UiTrambusTitle.vue'
+import UiRennesLogo from '../ui/UiRennesLogo.vue'
+import NetworkFigures from './NetworkFigures.vue'
+import { usePanelsStore } from '@/stores/panels'
+
+import { ChevronRightIcon } from '@heroicons/vue/20/solid'
+import { ChevronLeftIcon } from '@heroicons/vue/20/solid'
+
+const panelStore = usePanelsStore()
+function toggleInformationPanel() {
+  panelStore.toggleInformationPanel()
+}
+</script>
+
+<template>
+  <div class="flex flex-row shadow-sm">
+    <div
+      class="p-4 bg-white flex flex-col gap-2 max-w-full"
+      :class="{ hidden: !panelStore.isInformationPanelShown }"
```

Comment:
I think they will ask for an animation, therefore, using the `hidden` class will not work. You should play with size change transition, or maybe a translation.

Reply:
should I address the transition now with some basic transition or should we wait until the final decision? Based on the daily, they are still working on this transition.

Reply:
let's wait